conf/zeppelin-env.sh.template

```diff
@@ -76,3 +76,5 @@
 # export ZEPPELINHUB_API_TOKEN			# Refers to the Zeppelin instance token of the user
 # export ZEPPELINHUB_USER_KEY			# Optional, when using Zeppelin with authentication.
 
+#### Mahout configuration ####
+# export MAHOUT_HOME=               # Directory of Apache Mahout installation
```

Comment:
MAHOUT_HOME must be set here or in terp properties. 

Also spark.home and java.home must be set in environement. (precedent on SparkR)